prerequisites.py

```diff
@@ -231,12 +231,28 @@ def install_isolate():
              root, 0o640, group=cmsuser_grp)
 
 
+def build_frontend():
+    """This function downloads the required front-end dependencies
+    like jQuery, Bootstrap, and so on. It basically just runs:
+      $ npm install
+    """
+    assert_not_root()
+
+    print("===== Running \"npm install\" to get front-end dependencies")
+    os.chdir(os.path.join("cms", "server"))
+    os.system("npm install")
```

Comment:
You should use `subprocess.check_call()` with `cwd`. Otherwise errors are not checked.

Reply:
Done :+1: